wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -89,21 +90,7 @@ def explorer_breadcrumb(
         "url_name": url_name,
         "url_root_name": url_root_name,
         "use_next_template": use_next_template,
-    }
-
-
-@register.inclusion_tag("wagtailadmin/shared/move_breadcrumb.html", takes_context=True)
-def move_breadcrumb(context, page_to_move, viewed_page):
-    user = context["request"].user
-    cca = get_explorable_root_page(user)
-    if not cca:
-        return {"pages": Page.objects.none()}
-
-    return {
-        "pages": viewed_page.get_ancestors(inclusive=True)
-        .descendant_of(cca, inclusive=True)
-        .specific(),
-        "page_to_move_id": page_to_move.id,
```

Comment:
we will still be able to remove this as there should no longer be any more need for the `move_breadcrumb` template tag.